qa/workunits/dencoder/test_readable.py

```diff
@@ -0,0 +1,340 @@
+#!/usr/bin/env python3
```

Comment:
Is anobody spawning this script? In `main` there is:

```
$ grep -r  test_readable ../qa
../qa/workunits/dencoder/test-dencoder.sh:$mydir/test_readable.py $CEPH_MNT/client.0/tmp/ceph-object-corpus-master
```

Reply:
@rzarzynski i'm missed a backport of the test into reef and squid. (https://github.com/ceph/ceph/pull/56743)
I'm going to add it here